CCP_App/src/main/java/com/example/demo2/custom/exception/NoElementFountException.java

```diff
@@ -0,0 +1,21 @@
+/************************************************************************
+ *                                                                      *
+ *  DDDD     SSSS    AAA        Daten- und Systemtechnik Aachen GmbH    *
+ *  D   D   SS      A   A       Pascalstrasse 28                        *
+ *  D   D    SSS    AAAAA       52076 Aachen-Oberforstbach, Germany     *
+ *  D   D      SS   A   A       Telefon: +49 (0)2408 / 9492-0           *
+ *  DDDD    SSSS    A   A       Telefax: +49 (0)2408 / 9492-92          *
+ *                                                                      *
+ *                                                                      *
+ *  (c) Copyright by DSA - all rights reserved                          *
+ *                                                                      *
+ ************************************************************************
+ *
+ * Author : Riyaardh Adam
+ * Created: 2023/02/21 10:34
+ *
+ ***********************************************************************/
+package com.example.demo2.custom.exception;
+
+public class NoElementFountException extends RuntimeException {
```

Comment:
Spelling error on class
Consider if this should rather be a "checked" exception

Reply:
Javadoc please